dfusion_rust_core/src/models/auction_element.rs

```diff
@@ -0,0 +1,88 @@
+extern crate mock_it;
+
+use byteorder::{BigEndian, ByteOrder};
+
+use std::collections::HashMap;
+
+use web3::types::{H160, U256};
+
+use crate::models::{BatchInformation, Order};
+
+#[derive(Debug, Default, PartialEq)]
+pub struct AuctionElement {
```

Comment:
Why are we putting this in the shared library? Right now this object is still pretty private to the smart contract and not passed around at all. Might be worth while keeping this an implementation detail.

Reply:
I am really not sure where a good place for the auction_element is.
I agree with Felix that in the short term, it is a private object, hence it does not belong into dfusion_core.
Then I thought to put it into stablex_contract.rs, as it is only used there, but then the whole file was soo big and messy to read. Hence, I put it all into stable_contract_auction_element.rs. 
I think, it is the best solution, but I am not sure. So I am very interested into your opinions.
@fleupold @bh2smith